__tests__/support/sandbox.js

```diff
@@ -0,0 +1,13 @@
+const Hexo = require('hexo')
+const path = require('path')
+const {createSandbox} = require('hexo-test-utils')
+
+module.exports = function getSandbox() {
+  return createSandbox(Hexo, {
+    fixture_folder: path.join(__dirname, '..','fixtures'),
+    plugins: [
+      path.join(__dirname, '..', '..', 'node_modules', 'hexo-renderer-ejs', 'index.js'),
```

Comment:
This is not ideal, there should be way to pass just `hexo-renderer-ejs` eventually.